arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1082,6 +1082,9 @@
             "$ref": "#/definitions/EventHubProperties"
           }
         },
+        "routing": {
```

Comment:
Looks like this is an optional property. So if someone make a request to get an IotHub, 
 - how will this property look like? 
   - Will it not be present at all?
   - Will it be present with a null value ?
   - Will it be present with some of the properties (readOnly), set ?
   - Or will it be an empty object?